package-lock.json

```diff
@@ -1,1178 +1,8 @@
 {
   "name": "bicep-registry-modules",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Could we restore the package-lock file? The version change might be due to an old `npm` version. Upgrading `npm` to > v7 should solve the problem.